lib/public/css/home/highlight.scss

```diff
@@ -0,0 +1,109 @@
+.highlight {
+  background: $white;
+  color: $gray;
+  margin: 0;
+  text-align: center;
+
+  @media #{$xsmall-only} {
+    border-bottom: solid 2px $light-gray;
+  }
+
+  h2 {
+    font-size: $h2-font-size;
+    font-weight: 500;
+
+    @media #{$small-up} {
+      display: none;
+    }
+
+  }
+
+  p {
+    font-size: $paragraph-font-size;
+
+    @media #{$small-up} {
+      display: none;
+    }
+
+  }
+
+  &__wrapper {
+    @extend .wrapper;
+    padding: 1rem 0;
+
+    @media #{$small-up} {
+      border-bottom: solid 2px $light-gray;
+    }
+
+  }
+
+  &__grid {
+    display: grid;
+
+    @media #{$small-up} {
+      grid-gap: 10px;
+      grid-template-columns: 33% 33% 33%;
```

Comment:
Awesome, `grid`! You can use `fr` units instead of `%` so you don't have to do any math yourself. You can write `grid-template-columns: 1fr 1fr 1fr;`, or use a shortcut with `grid-template-columns: repeat(3, 1fr);`.

Also, since you're not using any grid features until after you hit the breakpoint, you don't need to set `display: grid` _until_ that breakpoint—you're probably fine with the default display settings at points smaller than that.

Grid is exciting, but it may not be necessary if you're only laying things out in one direction (then you can just use `flex`)—but here it looks like hitting View All Modules may expand this section and show more stuff, in which case `grid` is totally the right call!

Have fun! :)

Reply:
Thanks so much man! I'll definitely address these things. that's way better than percentages